app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Admin::OrdersController < Admin::AdminsController
+  def index
+    @title = t "status_order.open"
+    load_orders :open
+  end
+
+  def confirmed
+    @title = t "status_order.confirmed"
+    load_orders :confirmed
+  end
+
+  def shipping
+    @title = t "status_order.shipping"
+    load_orders :shipping
+  end
+
+  def completed
+    @title = t "status_order.completed"
+    load_orders :completed
+  end
+
+  def cancelled
+    @title = t "status_order.cancelled"
+    load_orders :cancelled
+  end
+
+  def show; end
+
+  private
+
+  def load_orders status
+    @orders = Order.send(status)
+                   .includes(:order_details)
+                   .recent_orders.page(params[:page])
+                   .per(Settings.show_10)
+    render "index"
```

Comment:
```suggestion
    render :index
```
Code này chị nghĩ xử lý ở pull khác